### PR TITLE
[Term Entry] AI Activation Function

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Taking one of the unclaimed AI Issues. This assumes that #2190 is merged.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/ai/concepts/neural-networks/terms/activation-function/activation-function.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Description: 'An activation function is the function used by a node in a neural network to take the summed weighted input to the node and transform it into the output value.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Deep Learning'
+  - 'Machine Learning'
+  - 'Neural Networks'
```
Comment (caupolicandiaz) [inline content/ai/concepts/neural-networks/terms/activation-function/activation-function.md:12]: perhaps cull a couple of these tags

File: content/ai/concepts/neural-networks/terms/activation-function/activation-function.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - 'Algorithms'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+An **activation function** is the function used by a node in a neural network to take the summed weighted input to the node and transform it into the output value. The output value can be fed into the next hidden layer or received from the output layer. For networks that engage in tasks of any significant complexity, the activation function needs to be non-linear.
+
+## Types of Activation Functions
```
Comment (caupolicandiaz) [inline content/ai/concepts/neural-networks/terms/activation-function/activation-function.md:22]: I believe the "relu" is one the most commonly used functions and probably should be mentioned here as well

**Comment by SSwiniarski** [comment]
@caupolicandiaz I addressed your comments.

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by KTom101** [comment]
Link: https://www.codecademy.com/resources/docs/ai/neural-networks/activation-function

**Pull request merged**